charts/shield/templates/host/_helpers.tpl

```diff
@@ -54,24 +54,44 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 
 {{- define "host.tolerations" -}}
-{{- $tollerations := concat .Values.tolerations .Values.host.tolerations -}}
-{{- with $tollerations -}}
+{{- $tolerations := concat .Values.tolerations .Values.host.tolerations -}}
+{{- with $tolerations -}}
 {{- . | toYaml -}}
 {{- end -}}
 {{- end -}}
 
 {{- define "host.affinity" -}}
 {{- $affinity := merge (dict) .Values.affinity .Values.host.affinity -}}
+{{- if $affinity }}
 {{- with $affinity -}}
 {{- . | toYaml -}}
 {{- end -}}
+{{- else -}}
+nodeAffinity:
```

Comment:
Why don't we set this as default in the values for `host.affinity`?

Reply:
There wasn't a strong reason not to. I was just exploring an option here to not clutter up the default `values.yaml` with verbose default settings. I'm open to moving this out to the file though. What are your thoughts?

Reply:
The only good point i have in my mind for having it in the values it that the generated documentation will report it as default value, while doing it in this way will hide it in the helpers

Reply:
That's a good point. 🤔 👍🏻

Reply:
I agree on hiding from the doc (but maybe we may have some annotation for that?) but adding in the default values looks like more clear to users, plus will make the code simpler here  (no if else, no special scenario, ..) 🤔